src/modules/search-time/selector/index.tsx

```diff
@@ -43,14 +43,52 @@ export default function SearchTimeSelector({
           };
 
     setSelectedMode(newState);
-
     onChange(newState);
   };
 
+  const addCurrentTimeToDate = (date: string) => {
+    return new Date(`${date} ${format(new Date(), 'HH:mm:ss')}`);
+  };
+
+  const resetToCurrentTime = () => {
+    setSelectedTime(() => format(new Date(), 'HH:mm'));
+  };
+
+  const resetToCurrentDate = () => {
+    setSelectedDate(new Date());
+  };
+
+  const addCurrentDateToTime = (time: string) => {
+    const [hours, minutes] = time.split(':').map(Number);
+    const now = new Date();
+    now.setHours(hours, minutes, 0, 0);
+    return now;
+  };
+
+  const isPastDate = (selectedDate: string) => {
```

Comment:
I think this exists in date utils from before?

Reply:
I'm not entirely sure what you are referring to as already existing here. Do you mean the isPastDate function?

Reply:
There's a function in `src/utils/date.ts`, `isInPast()`, that might work as you want here.

Reply:
I tried to use isInPast() instead of  isPastDate(). This is what i found out: 
The event.target.value is received on the format 'yyyy-MM-dd'. When using isInPast(), this string is converted to date format. Due to some milliseconds that is being added during the conversion, the current date is regarded as a past date. 

We can use the isInPast() if we want to, but this still requires some additional logic to ensure that the current date is not view as a past date. I therefore suggest that we keep using the isPastDate().